static/app/views/performance/transactionSummary/transactionTags/content.tsx

```diff
@@ -181,6 +182,8 @@ function InnerContent(
 
   const query = decodeScalar(location.query.query, '');
 
+  const projectIds = useMemo(() => eventView.project?.slice(), [eventView.project]);
```

Comment:
nit: we should just pass in the projects as a prop to `TransactionSearchQueryBuilder` and memoize the ids within the component so we don't have to duplicate this line each time we use the component

Reply:
the reason this is here is due to the incompatible types in `EventView.project` which is `readonly number[]` while `PageFilter['projects']` has a type of `number[]`.

Reply:
ah gotcha. can we not just cast it as `number[]` instead of cloning the array?